ops/copy.encrypted

```diff
@@ -0,0 +1,18 @@
+#! /bin/bash -
```

Comment:
I totally understand why we have it and why we need it.  But its an annoying step, so I'm trying to think of alternatives.

Throwing this out there...
What if we no longer store the vars with a `.encrypted` extension, and instead keep them as-is (like we do in `bcda-ops`)?  Instead, we create a directory called `encrypted_config` and use the directory name to key off of for managing secrets, instead of extension?  (Or, we just decide that `config` is where we keep all encrypted vars and keep it named as-is).
Then, the encryption/decryption process happens to the file in-place, just like in `bcda-ops`.  No need for this script, and our `pre-commit` hook will ensure we don't commit unencrypted vars.

Reply:
A separate directory might slightly reduce confusion, but we would still need a version of this script (at least for Travis) to copy the files to their destination once they've been unencrypted.  

As for having a separate step: it's totally possible to embed this in the `secrets` script with a `--copy` argument.  But as I was working on implementing that, it became clear that the `secrets` script is more general purpose (it doesn't care what files it's working on or how they're named), and this script cares about naming (or if we follow your suggestion, about knowing a destination).

Do you like it better if we remove this script and create:
- A dedicated `shared_files/config/encrypted/` subdirectory
- An enhancement to the `secrets` script that allows for `ops/secrets --decrypt shared_files/config/encrypted/ --copy shared_files/config/`

Reply:
> but we would still need a version of this script (at least for Travis) to copy the files to their destination once they've been unencrypted.
Wouldn't the `copy` that Travis needs just be a one-liner in the `.travis.yml` file?

Reply:
> A dedicated `shared_files/config/encrypted/` subdirectory

We could keep it named as is `/shared_files/config` and update the `README` within that directory to indicate that only encrypted data lives in this directory.  Then, the env vars could be named exactly what we need them to be and no unnecessary copying is required.

> An enhancement to the secrets script that allows for `ops/secrets --decrypt shared_files/config/encrypted/ --copy shared_files/config/`

I don't think we need to change the `secrets` script.

Reply:
OK, I think the latest commits following your suggestion make for a better workflow.